documentation/docs/pages/index.md

```diff
@@ -1,3 +1,93 @@
 
-Welcome to ...
+# Home
+
+Welcome to the documentation for the [ACCESS-ESM1.6 model configurations](https://github.com/ACCESS-NRI/access-ESM1.6-configs)! 
+
+
+## ACCESS-OM3 Documentation Overview
+
+See the navigation links on the left. Some reading tips, see:
+
+ - [Contributing](/contributing) 
+ - [Inputs](/inputs/Forcing-data-models) 
+ - [Configuration choices/Configurations](/configurations/Overview/) 
+ - [Infrastructure](/infrastructure/Architecture/) 
+
+## ACCESS-OM3-Configs Overview
+ACCESS-ESM1.6 configurations are provided via branches in the [access-esm1.6-configs](https://github.com/ACCESS-NRI/access-esm1.6-configs) GitHub repository. The [access-esm1.6-configs](https://github.com/ACCESS-NRI/access-esm1.6-configs) repository contains several configurations using the following components:
+
+- [MOM5](https://github.com/ACCESS-NRI/mom5) ocean model
+
+All the configurations use the [Payu](https://payu.readthedocs.io/en/latest/) workflow management tool, and pre-built executables available on [NCI](https://nci.org.au/).
+
+### Repository structure
+
+The [`main`](https://github.com/ACCESS-NRI/access-esm1.6-configs/tree/main) branch does not store any model configurations, only documentation.
+
+Each configuration in [github.com/ACCESS-NRI/access-esm1.6-configs](https://github.com/ACCESS-NRI/access-esm1.6-configs) repository is stored as a git branch. Most of the branches are named according to the following naming scheme:
+
+`{release}-{scenario}[+{modifier}]`
```

Comment:
```suggestion
`release-{scenario}[+{modifier}]`
```
"release" is not a placeholder for something else here like `{scenario}` and `{modifier}`